dashboard/app/models/contact_rollups_pardot_memory.rb

```diff
@@ -112,25 +112,38 @@ def self.download_deleted_pardot_prospects(last_id = nil, limit = nil)
     end
   end
 
-  def self.create_new_pardot_prospects
+  def self.create_new_pardot_prospects(dry_run: false)
```

Comment:
nit rename `dry_run` to something such as `is_dry_run` to imply it's a boolean

Reply:
Done!